### PR TITLE
Rework overview-tool bar so it remains a single row.

### PR DESCRIPTION
**Pull request by sg00dwin**
toolbar with [project status toggle pr](https://github.com/openshift/console/pull/745)

overview-toolbar

### CODE REVIEW

**Comment by spadgett** [comment]
FYI, this will conflict with #745 depending on who merges first

File: frontend/public/components/overview/_overview.scss [hunk 1]
```diff
@@ -2,27 +2,77 @@
...
+    margin-bottom: 0;
+    padding-left: 0;
+  }
+
+  &:last-child {
+    border-left: 1px solid $color-pf-black-300;
+    padding-left: 8px;
+  }
+}
+@media (min-width: $screen-xs-min) {
```
[the remark this answers is not in the document]
Comment (sg00dwin) [reply frontend/public/components/overview/_overview.scss]: agreed and updated

File: frontend/public/components/overview/_overview.scss [hunk 2]
```diff
@@ -2,27 +2,75 @@
...
 
+.overview-toolbar__form-group {
+  flex: 1 1 50%;
+  min-width: 0; // enable truncation
+  padding-right: 8px;
+  @media (min-width: $screen-xs-min) {
+    padding-right: 15px;
+  }
+
+
```
Comment (rhamilto) [inline frontend/public/components/overview/_overview.scss]: Nit:  extra line

Comment (sg00dwin) [reply frontend/public/components/overview/_overview.scss]: removed :)

**Comment by sg00dwin** [comment]
PR rebased with https://github.com/openshift/console/pull/745
Screen recording updated

**Pull request merged**